### DISCUSSION TITLE
Do you have any plan for eyecandy things?

### DISCUSSION THREAD

**Discussion by gjshinji** [pinnacle-comp/pinnacle#218] [Ideas]
AwesomeWM implemented animations through scripts like [rubato](https://github.com/andOrlando/rubato) or compositor like picom. Does your plan have something like this? I'm definitely waiting for a compositor to come along that could be an alternative to hyprland...

**Comment by Ottatop** [comment]
Stuff like animations and blur are *planned* but they're not a current priority for v0.1. The focus right now is to make a compositor that is somewhat stable and implements most of the things that Wayland compositors are expected to have.
